sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosTableSchemaInferrer.scala

```diff
@@ -211,7 +211,16 @@ private object CosmosTableSchemaInferrer
 
   // scalastyle:on
   private def inferDataTypeFromArrayNode(node: ArrayNode, allowNullForInferredProperties: Boolean): Option[DataType] = {
-    Option(node.get(0)).map(firstElement => inferDataTypeFromJsonNode(firstElement, allowNullForInferredProperties))
+    val notNullElements = node.elements.asScala
+      .filter(element  => !element.isNull)
```

Comment:
NIT: my preference would be to have LF before each .
like 
node
  .elements
  .asScala
  .filter(xxx)